app/src/main/AndroidManifest.xml

```diff
@@ -46,6 +46,13 @@
                 android:resource="@xml/button_widget_info" />
         </receiver>
 
+        <!-- Start things like SensorWorker on device boot -->
+        <receiver android:name=".HomeAssistantBootReceiver">
+            <intent-filter>
+                <action android:name="android.intent.action.BOOT_COMPLETED"/>
```

Comment:
If we are going to add a generic boot receiver then we should remove the boot specific logic from `LocationBroadcastReceiver` and send a broadcast to it instead starting location tracking.

Reply:
I can look into this change!